lib/myxql/client.ex

```diff
@@ -203,11 +207,39 @@ defmodule MyXQL.Client do
     end
   end
 
+  # If the packet size equals max packet size, save the payload, receive
+  # more data and try again
+  defp recv_packets(
+         <<size::uint3, _seq::uint1, payload::string(size), rest::binary>>,
+         decoder,
+         decoder_state,
+         timeout,
+         client,
+         partial
+       )
+       when size == @default_max_packet_size do
```

Comment:
what do you think about:

```suggestion
       when size >= @default_max_packet_size do
```

and changing `size != @default_max_packet_size` to `size < @default_max_packet_size` accordingly above and in potential other places?

Now, it's not possible to put more than the default max packet size (2^24 - 1) because on the protocol we have 3 bytes for the packet size, so perhaps what I'm proposing is considered too defensive. WDYT?

Reply:
No, you are right - your suggestion is potentially more future-proof. Fix incoming.

Reply:
https://github.com/elixir-ecto/myxql/pull/105/commits/9b72bca0b0a942a97a7cea922bca71be776f1c06

Reply:
Gah, forgot about adjusting the other clause, here it is https://github.com/elixir-ecto/myxql/pull/105/commits/a56925d8565d190332c06efd4f128a2a977f65c9